cypress/integration/individual-pages/activity.spec.ts

```diff
@@ -74,11 +58,28 @@ describe("As a beacon owner, I want to submit the primary activity for my beacon
     thenMyFocusMovesTo("#otherActivityText");
   });
 
-  it("routes to the next page if there are no errors with Other pleasure vessel selected", () => {
-    givenIHaveSelected(otherActivitySelector);
+  it("does not show errors if valid input is given to Other activity", () => {
+    givenIAmAMaritimePleasureUser();
+    andIHaveSelected(otherActivitySelector);
     whenIType("Surfboard", "#otherActivityText");
     whenIClickContinue();
+    thenThereAreNoErrors();
+  });
 
+  it("routes to the next page if there are no errors with Other pleasure vessel selected", () => {
```

Comment:
@zackads Can remove these two tests (this one and one above) as we're testing this scenario within the other multi-step tests below

Reply:
Done ✅